src/inttest/java/com/faforever/api/leaderboard/LeaderboardControllerTest.java

```diff
@@ -23,7 +23,7 @@ public void ladder1v1All() throws Exception {
 
     mockMvc.perform(asyncDispatch(mvcResult))
       .andExpect(status().isOk())
-      .andExpect(content().string("{\"data\":[{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"1\",\"attributes\":{\"name\":\"USER\",\"mean\":1500.0,\"deviation\":120.0,\"numGames\":5,\"wonGames\":1,\"rank\":1,\"rating\":1140}},{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"2\",\"attributes\":{\"name\":\"MODERATOR\",\"mean\":1200.0,\"deviation\":90.0,\"numGames\":5,\"wonGames\":2,\"rank\":2,\"rating\":930}},{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"3\",\"attributes\":{\"name\":\"ADMIN\",\"mean\":1000.0,\"deviation\":100.0,\"numGames\":5,\"wonGames\":3,\"rank\":3,\"rating\":700}}]}"));
+      .andExpect(content().string("{\"data\":[{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"1\",\"attributes\":{\"name\":\"USER\",\"mean\":1500.0,\"deviation\":120.0,\"numGames\":5,\"wonGames\":1,\"rank\":1,\"rating\":1140}},{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"2\",\"attributes\":{\"name\":\"MODERATOR\",\"mean\":1200.0,\"deviation\":90.0,\"numGames\":5,\"wonGames\":2,\"rank\":2,\"rating\":930}},{\"type\":\"ladder1v1LeaderboardEntry\",\"id\":\"3\",\"attributes\":{\"name\":\"ADMIN\",\"mean\":1000.0,\"deviation\":100.0,\"numGames\":5,\"wonGames\":3,\"rank\":3,\"rating\":700}}],\"meta\":{\"totalPages\":1,\"totalRecords\":3}}"));
```

Comment:
Don't assert json via String. The order should not take any role.
Check MockMvcResultMatchers.jsonPath instead to check each property.
(Applies to all tests)